src/Application/UI/Presenter.php

```diff
@@ -213,6 +216,7 @@ public function run(Application\Request $request): Application\IResponse
 
 			// RENDERING VIEW
 			$this->beforeRender();
+			$this->onRender($this);
```

Comment:
Method is missing IResponse declared in phpdoc.
View and render method parameters could be useful.
If onRender, why not onAction?

Reply:
PHPDoc: my bad, i writed it directly on GH.. will fix it...
I want onBeforeRender and onAfterRender, but after i spoke with DG, onRender is better, as onAfterRender have almost no usage...